versions.yaml

```diff
@@ -1,9 +1,9 @@
 grpcui: 1.3.1
-jsonnetfmt: 4906958414ed9617e3fe5acac5752f6f8426551c # use a tagged version whenever they release past v0.18.0 (https://github.com/google/go-jsonnet)
-goimports: 0.1.12
-delve: 1.9.1
-gotestsum: 1.7.0
-lintroller: 1.15.1
+jsonnetfmt: v1.19.1
```

Comment:
No `v` pls

Reply:
(might need to go fix jsonnetfmt code to support no `v`, but the convention is no `v` here)

Reply:
Seems like it works without the v.